assets/js/admin.js

```diff
@@ -9,6 +9,11 @@
 		$( '#date-submit' ).trigger( 'click' )
 	});
 
+	//Bulk verify alert
+	$( '#doaction' ).click(function (e) {
+		alert( 'Are you sure?' );
```

Comment:
@eriktad is there a way to say "No" and stop the process? I guess you must use confirmation box, not alter.

Reply:
Fixed